tests/config.json

```diff
@@ -34,20 +34,21 @@
                 "type": "service",
                 "account": "md-ingestion"
             },
-            "topic": "backbeat-test-ingestion",
+            "topic": "backbeat-ingestion",
             "zookeeperPath": "/ingestion",
+            "cronRule": "*/5 * * * * *",
             "sources": [
                 {
-                    "name": "ingest-site-1",
+                    "name": "zenkobucket",
```

Comment:
We no longer get this config from `config.json`. Does it make sense to still rely on this?

Reply:
https://github.com/scality/backbeat/blob/485f64744a0e17b5d1dbde95227ebb4c80440162/lib/queuePopulator/IngestionReader.js#L31

I did it so I could create a simpler producer used in testing

Reply:
For pause/resume, I created tests that mocks out IngestionPopulator's MongoClient (for fetching all ingestion enabled buckets), and the Config event emitter for getting updates on storage locations.

I think it would be very hard to build tests w/out mocking this out. Based off how pause/resume went, I can do the IngestionPopulator functional tests similarly where I mock similar functions, and maybe refactor so we have mock helper classes for this stuff?

Also, in IngestionReader, with decrypted location secret, I feel we should mock this out (like how Jay did it in this PR) because it relies on an ssh key generated in Orbit world. Ultimately, all we care about is passing the "verySecretKey1" to the IngestionProducer class as the secret key so we can continue using the CI in-memory cloudserver client.